backend/include/sdk.hpp

```diff
@@ -53,6 +54,24 @@ class SDK {
     void handleAFVEventForWebsocket(sdk::types::Event event,
         const std::optional<std::string>& callsign, const std::optional<int>& frequencyHz);
 
+    /**
+     * @brief Builds a JSON object for the station state.
+     *
+     * @param callsign The callsign of the station. Optional.
+     * @param frequencyHz The frequency of the station.
+     *
+     * @return The JSON object for the station state.
+     */
+    nlohmann::json SDK::buildStationStateJson(
```

Comment:
Split out into its own method so the result can be sent both via websocket and IPC. I'm not super wild about this living in SDK but I couldn't think of a better place to put it.